public/styles/config.css

```
@@ -0,0 +1,239 @@
+.config-module {
+  display: block;
+  break-inside: avoid;
+  margin-bottom: 38px;
+  background: rgba(32, 28, 50, 0.92);
+  border-radius: 16px;
+  box-shadow: 0 2px 18px 0 rgba(32, 28, 50, 0.13);
+  padding: 28px 32px 22px 32px;
+  width: 100%;
+  max-width: 480px;
+  min-width: 320px;
+  display: flex;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

display is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)